public/index.html

```diff
@@ -33,19 +33,22 @@ <h1>Base<sub>x</sub> conversor</h1>
         </p>
         <div id="bin">
           <input
+            id="bin2Dec"
             type="button"
-            value="Decimal"
-            onclick="bin2Dec(document.getElementById('num').value)"
+            value="decimal"
+            onclick="changeNumber(document.getElementById('bin2Dec').id)"
```

Comment:
Every time you call this function: `changeNumber(...)`, why do you make the template get the value of the input? Why not doing:

`onclick="changeNumber('bin2Dec')"` 
`onclick="changeNumber('bin2Hex')"` 
`onclick="changeNumber('bin2Oct')"`

Reply:
In the end this is what arrives to the `changeNumber` function and anyway you know this value beforehand when you're defining the `onclick` listener, there's no reason to grab a value from the DOM for this.

Reply:
I made it this way I wanted to use the id of the html, but yes, I think is a better way just calling the function with the needed value.

Reply:
What do you think are the benefits of using the ID of the input as a parameter?

Reply:
none, it was there because I was using the value tag before and only changed the tag to id. I corrected that
https://github.com/GonzoVM/bin2dec/commit/3574b7f43e64689a41aa2fe0d3adce17e2910126

Reply:
good job, it looks better now!